google-cloud-bigquery/README.md

```diff
@@ -214,6 +214,10 @@ Troubleshooting
 
 To get help, follow the instructions in the [shared Troubleshooting document](https://github.com/GoogleCloudPlatform/gcloud-common/blob/master/troubleshooting/readme.md#troubleshooting).
 
+Transport
+---------
+BigQuery uses HTTP transport.
```

Comment:
-> "BigQuery uses HTTP for the transport layer." (Use the pattern in the rest of the services too)